### DISCUSSION TITLE
Como escolher a SDK ao começar um projeto no VS Code?

### DISCUSSION THREAD

**Discussion by Matheus-95** [balta-io/2801#95] [Perguntas e Respostas] [answered]
Olá!

Este é meu primeiro mês estudando lógica de programação / C#.

Gostaria de entender como eu consigo iniciar um projeto em .NET Core 3.1 no VS Code usando CLI.

A situação é a seguinte:

Instalei primeiramente o .NET 6 enquanto assistia o curso "Fundamentos do C#", porém quando fui acompanhar o vídeo do Balta, vi que o meu arquivo Program.cs estava com apenas a linha de código: "Console.WriteLine("Hello, World!");".

Sendo que no curso o arquivo Program.cs dele estava com a seguinte estrutura:

using System;

namespace MeuApp
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World");
        }
    }
}

No final das contas acabei descobrindo que é porque o meu projeto foi iniciado em .NET 6 e o que ele utilizou no curso foi o .NET Core 3.1, então acabei fazendo algumas alterações na mão mesmo, criando um arquivo .json e dizendo que o que eu utilizaria seria o .NET Core 3.1.

Ai que está minha dúvida, quando eu tenho mais de um SDK instalado, como eu consigo dizer ao VS Code que quero começar um projeto em .NET Core 3.1 especificamente? Sem que ele puxe a última versão instalada como default.

Espero que não tenha ficado confusa a minha dúvida. 

Desde já agradeço pela paciência e atenção! 👍

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Matheus-95, consegue sim (Recomendo usar a mesma versão do Curso... .NET 5)
```
dotnet new console -o NomeDoProjeto -f net5.0
```

No caso o **-f net5.0** especifica a versão que ele vai criar o projeto... poderia ser **-f net 3.1**.

**Reply by Matheus-95** [reply]
Muito obrigado @andrebaltieri !!!
